File: Programming Fundamentals with Python/Functions - Exercise/04_odd_and_even_sum.py
def odd_even(number):
    odd_sum = 0
    even_sum = 0
    for i in range(len(number)):
        if int(number[i]) % 2 == 0:
            even_sum += int(number[i])
        else:
            odd_sum += int(number[i])
    return f'Odd sum = {odd_sum}, Even sum = {even_sum}'


num = input()

print(odd_even(num))

File: JS Front-End/Functions and Statements - Exercises/04.oddAndEvenSum.js
function oddAndEvenSum(number) {
  let numberAsString = String(number);
  let oddSum = 0;
  let evenSum = 0;
  for (let index = 0; index < numberAsString.length; index++) {
    let currentNum = Number(numberAsString[index]);
    if (currentNum % 2 === 0) {
      evenSum += currentNum;
    } else {
      oddSum += currentNum;
    }
  }
  console.log(`Odd sum = ${oddSum}, Even sum = ${evenSum}`);
}